test/assets/quality_gates_shipyard.yaml

```diff
@@ -0,0 +1,31 @@
+apiVersion: "spec.keptn.sh/0.2.0"
```

Comment:
this file is not needed for this PR, right?

Reply:
true, this is not needed for the PR. I have removed it